fdbclient/SystemData.h

```diff
@@ -170,37 +189,53 @@ ProcessClass decodeProcessClassValue( ValueRef const& );
 UID decodeProcessClassKeyOld( KeyRef const& key );
 
 //   "\xff/conf/[[option]]" := "value"
+//	An umbrella prefix for options mostly used by the DatabaseConfiguration class.
+//	See DatabaseConfiguration.cpp ::setInternal for more examples.
 extern const KeyRangeRef configKeys;
 extern const KeyRef configKeysPrefix;
 
+//	The differences between excluded and failed can be found in "command-line-interface.rst"
+//	and in the help message of the fdbcli command "exclude".
+
 //   "\xff/conf/excluded/1.2.3.4" := ""
 //   "\xff/conf/excluded/1.2.3.4:4000" := ""
 //   These are inside configKeysPrefix since they represent a form of configuration and they are convenient
 //   to track in the same way by the tlog and recovery process, but they are ignored by the DatabaseConfiguration
 //   class.
+//	 The existence of an empty string as a value signifies that the provided IP has been excluded.
```

Comment:
Can you also explain the meaning when the value is "" only?

Reply:
Isn't that what I've done here? The empty string is just "", unless I'm misunderstanding something here.